package-lock.json

```diff
@@ -7147,7 +7147,7 @@
     },
     "external-editor": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
```

Comment:
Can you guys revert this etc so it stays https instead of just http?

Reply:
Good catch, fixed. Thanks!